### ISSUE TITLE
timetable editor: does not take in account pattern travel time

### ISSUE THREAD

**Issue by skinkie** [catalogueglobal/datatools-ui#216]
![image](https://user-images.githubusercontent.com/502394/42622263-46372370-85c0-11e8-953b-ce9735a744d9.png)

I would expect that in the timetable editor I only would need to provide the departure time. But the Travel Time is not automatically added:

![image](https://user-images.githubusercontent.com/502394/42622327-72409ece-85c0-11e8-9693-ad2b79b391f8.png)

**Comment by landonreed** [comment]
I believe you must have created the trip when the pattern had one stop, then added Vlissingen, revisited the timetable editor, and found the stop blank.

Currently this is the expected behavior, but we do recognize that the ideal behavior would be to apply the travel times to existing trips.

**Comment by skinkie** [comment]
No, the pattern was already created. I'll invite you to the project, so you can see for yourself :)

**Comment by landonreed** [comment]
Do you mean the pattern already had two stops before creating the trop `0612`? I don't think this is the case. If you add a new trip, you should see the travel/dwell time for the stop automatically applied here.

**Comment by skinkie** [comment]
This is what happens if you add a trip to the end:

![image](https://user-images.githubusercontent.com/502394/42639135-97f4526c-85ef-11e8-874a-8fdecc139463.png)

Editing that:
![image](https://user-images.githubusercontent.com/502394/42639171-a9964868-85ef-11e8-9be4-322895585c36.png)

So there are quite a view advancements that could be taken here. It should obviously automatically fill in the travel time. But also adding the trip might be done more intuitively. For example by providing the interval time ahead of time in a box next to the "add" button. Additionally it would be a nice feature if you could increase/descrease the time without typing.

**Comment by landonreed** [comment]
See #172.

**Comment by skinkie** [comment]
I would like to see that one work. But I hope after typing 21:00:00 it would just update (untouched) fields.

**Comment by landonreed** [comment]
Currently, the way to offset all times for a trip is to use the "offset" feature.

**Comment by skinkie** [comment]
I understand, I think the "excel" way of just entering a bunch of times, and the other columns follow, is far more practical.
